store/interfaces.json

```diff
@@ -107,6 +107,6 @@
     "component": "*"
   },
   "info-card": {
-    "component": "*"
+    "component": "vtex.store-components/InfoCard"
```

Comment:
This interface must be declared in you store-components, like product-name, product-price and so on...

Reply:
bruno abreu told me to this way, but now I can see that the other way is better and works.